landing/index.html

```diff
@@ -1,1043 +1,1146 @@
-                           </div> 
-                        </div>
-                     </div>
-                     
-                     
-                     <div>
-                        <div class="li">
-                           <div class="photo"><img src="images/u1.png" alt="user-pic"></div>
-                           <div class="slide1">
-                              
-                              <p>I have gotten at least 50 times the value from End of coffee. End of coffee has really helped our business. </p>
-                              <span>- Devil Jhon</span>
-                           </div> 
-                        </div>
-                     </div>
-                     
-                     <div>
-                        <div class="li">
-                           <div class="photo"><img src="images/u2.png" alt="user-pic"></div>
-                           <div class="slide1">
-                              
-                              <p>Not able to tell you how happy I am with End of coffee. I will let my mum know about this, she could really make use of End of coffee!</p>
-                              <span>- Simmy Ilona</span>
-                           </div> 
-                        </div>
-                     </div>
-                     
-                     <div>
-                        <div class="li">
-                           <div class="photo"><img src="images/u3.png" alt="user-pic"></div>
-                           <div class="slide1">
-                              
-                              <p>I will recommend you to my colleagues. Dude, your stuff is the bomb! Since I invested in End of coffee I made over 100,000 dollars profits.</p>
-                              <span>- John Doe</span>
-                           </div> 
-                        </div>
-                     </div>
-                     
-                     
-                     
-                  </section> 
-                  
-               </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- Description End -->
+
+    <div class="screen-section dark-section dark-color-bg" id="screens">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-12 col-md-12 col-sm-12">
+            <div class="heading">
+              <span>Previews</span>
+              <h2>Screenshot in details</h2>
+            </div>
+          </div>
+        </div>
+        <div class="pro_det_tab_main">
+          <div id="parentHorizontalTab">
+            <ul class="resp-tabs-list hor_1">
+              <li>Dashboard</li>
+              <li>List</li>
+              <li>Cohort</li>
+            </ul>
+            <div class="resp-tabs-container hor_1">
+              <div>
+                <p><!-- vertical Tabs --></p>
+                <img
+                  class="tab-screen-image"
+                  src="images/screen-tab-1.png"

```

Comment:
what do you think about inserting current views of eoc?

Reply:
First, we need change the layout of app to match landing layout - than we can insert screens